From 5f5ebec7962c197fb8684ec2e3ef187c6ca8afc6 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Mon, 14 Nov 2016 20:15:47 +0800
Subject: [PATCH] UPSTREAM: ARM: rockchip: enable support for RV1108 SoC

Add a rockchip,rv1108 compatible.

Change-Id: I2ef5fc82cc0867c93be47e01fbb5fb2fc5fff451
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
[rk1108->rv1108 rename]
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
(cherry picked from commit 51f6bfc78beaaa4c42bd2ab394e2550285caab0c)
---
 arch/arm/mach-rockchip/rockchip.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-rockchip/rockchip.c b/arch/arm/mach-rockchip/rockchip.c
index 5e62077c03c9..a86f2efe4c46 100644
--- a/arch/arm/mach-rockchip/rockchip.c
+++ b/arch/arm/mach-rockchip/rockchip.c
@@ -69,6 +69,7 @@ static const char * const rockchip_board_dt_compat[] = {
 	"rockchip,rk3066b",
 	"rockchip,rk3188",
 	"rockchip,rk3288",
+	"rockchip,rv1108",
 	NULL,
 };
 
-- 
2.35.3

